src/components/vault/vault.jsx

```diff
@@ -598,7 +598,7 @@ class Vault extends Component {
     const { basedOn } = this.state
     const initialApy = '0.00'
 
-    if(asset && asset.stats) {
+    if(asset && asset.stats && asset.stats.apyOneWeekSample) {
```

Comment:
Right now `apyOneWeekSample` will fall back to `apyInceptionSample` if there is not enough data for the specified timeframe. This means that currently `apyOneWeekSample` will be greater than zero once harvest is called. This means that as soon as harvest is called this check will pass, which is probably not the intention.

Some people have asked me to set APY samples to null if there is not enough data for a given timeframe. If I were to do that, that would mean that we would never show vault stats until one week's worth of data has been collected. Thus, inception APY would show as '0.00' until one week's worth of data has been collected.

If the desired behavior is to prevent any vault APY from being shown until one week's worth of data is collected then this change won't work as intended until I update the API to not fall back to inception APY (I can do this soon!)